src/components/testimonial/_testimonial.scss

```diff
@@ -1,59 +1,58 @@
 .testimonial {
-
   &--inline {
     align-items: center;
     display: flex;
   }
 
   &__text {
     color: $color-grey-dark;
-    font-size: 1.3em;
-    margin: 0 0 .769em;
+    font-size: 1.3rem;
+    margin: 0 0 $spacing-3;
 
     .testimonial--inline & {
-      border: 1px solid $color-grey-light;
+      border: $border-width-base $border-style $color-grey-light;
       border-radius: $border-radius;
       color: $color-grey;
-      font-size: .850em;
-      margin: 0 $spacing-horizontal-lg 0 0;
+      font-size: .85rem;
+      margin: 0 $spacing-6 0 0;
       max-width: 20em;
       padding: 1.765em;
       position: relative;
 
       &::after {
         background-color: $color-white;
-        border: 1px solid $color-grey-light;
+        border: $border-width-base $border-style $color-grey-light;
         border-bottom-color: $color-white;
         border-left-color: $color-white;
         border-top-right-radius: $border-radius;
         content: '';
         display: block;
-        height: 1.176em;
+        height: 1.175em;
```

Comment:
- [ ] Can the `em` units in this selector be changed to `rem` equivalents?
- [ ] Are `width` and `height` linked? If so, consider a local variable.